apps/package.json

```diff
@@ -233,7 +233,7 @@
     "@codemirror/matchbrackets": "^0.18.0",
     "@codemirror/rectangular-selection": "^0.18.0",
     "@codemirror/search": "^0.18.0",
-    "@codemirror/state": "^0.18.0",
+    "@codemirror/state": "^0.18.7",
```

Comment:
0.18.7 is where a change that prevents copy/pasting into the editor was introduced:

https://github.com/codemirror/view/commit/b06ae0399118f2ad2292d131f524b289e2acb090

Reply:
🎉